model/build.gradle

```diff
@@ -3,12 +3,13 @@ apply plugin: 'com.google.protobuf'
 
 protobuf {
   protoc {
+    artifact = 'com.google.protobuf:protoc:3.14.0:osx-x86_64'
```

Comment:
I just started the CI run, so we'll need to see if it works, but will this artifact actually work non-OS X platforms?

Reply:
> yeah  i have updated the build bazel

Reply:
@BenHenning i have updated the build bazel

Reply:
I don't think that addresses my question @Smarshal21. It seems like this updated Gradle file won't work for non-Linux platforms (which is completely separate from the Bazel builds).